desk/src/components/desk/global/CustomIcons.vue

```diff
@@ -1220,23 +1220,25 @@
 			/>
 		</svg>
 		<svg
-			v-if="this.name == 'add-response'"
+			v-if="this.name=='add-response'"
 			:class="this.class"
-			:width="this.width"
-			:height="this.height"
-			viewBox="0 0 17 17"
+			:width="16"
+			:height="15"
```

Comment:
This might cause problems when used with different widths and heights. Only this icon shouldn't have static width.